src/Command/Taxonomy/TaxoDeleteAllCommand.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\Console\Command\Taxonomy;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command as BaseCommand;
+use Drupal\Console\Command\Shared\ContainerAwareCommandTrait;
+use Drupal\Console\Style\DrupalStyle;
+use Symfony\Component\Console\Input\InputArgument;
+
+/**
+ * Class TaxoDeleteCommand.
+ *
+ * @package Drupal\eco_migrate
+ */
+class TaxoDeleteAllCommand extends BaseCommand {
```

Comment:
Update class name from `TaxoDeleteAllCommand` to `DeleteAllCommand` since the command is already on a directory `Taxonomy`

You can see how the status commands are names on this image:
![namespace-taxonomy](https://cloud.githubusercontent.com/assets/366275/17317097/dfa5f5fc-582e-11e6-87c6-58162eb335c4.png)